pieforproviders/pieforproviders
# [PIE-2121] Fix multiple api calls in dashboard

## 💅🏼 What issue does this fix?
Fixes #2121

## 🍂 Before You Submit

* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?

## 🧳 Steps to test
- Open developer tools and go to network tab
- Load dashboard, which should then call case_list_for_dashboard once
- Choose a different month from the dropdown list and the endpoint is called again with the selected month as the query param

## 🕯 Caveats, concerns
The test for this fix hasn't been written. I wanted to get feedback on my approach before going forward since I'm not 100% sure about it.
The plan was to implement something like this in the tests for DashboardTitle:
```
it('makes a call to get dashboard data by selected month', () => {
    // Check for text in the dropdown, e.g. Mar. 2022
    // Click on the dropdown
    // Select a different month
    // Check for text, e.g. Feb. 2022 
    // Check if /case_list_for_dashboard has been called
  })
```
